src/AppBundle/Controller/DefaultController.php

```diff
@@ -95,6 +96,58 @@ public function newPostAction(Request $request)
     }
     
     /**
+     * @Route("/downvote", name="new_downvote")
+     * @Method({"POST"})
+     */
+    public function newDownvoteAction(Request $request) 
+    {
+        // Get the post_id
+        $post_id = $request->get("$post_id");
```

Comment:
Need to get $post_id from the POST request. 

Should look something like $request->get('post_id');